src/main/java/stsjorbsmod/powers/ShrapnelBloomPower.java

```diff
@@ -0,0 +1,93 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.patches.SelfExertField;
+
+import static stsjorbsmod.JorbsMod.JorbsCardTags.LEGENDARY;
+
+public class ShrapnelBloomPower extends CustomJorbsModPower implements OnDamageToRedirectSubscriber {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(ShrapnelBloomPower.class);
+    public static final String POWER_ID = STATIC.ID;
+    private int numPlays;
+
+    public ShrapnelBloomPower(final AbstractCreature owner, final int amount, int numPlays) {
+        super(STATIC);
+
+        this.owner = owner;
+        this.amount = amount;
+        this.numPlays = numPlays;
+
+        updateDescription();
+    }
+
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        if (!card.purgeOnUse && this.amount > 0 && !card.hasTag(LEGENDARY)) {
+            this.flash();
+
+            action.exhaustCard = true; // this is what corruption does
+            SelfExertField.selfExert.set(card, true);
+
+            AbstractMonster m = null;
```

Comment:
You can simplify L42-L60 a bit similar to what we've already done for `PlayNextAttackThisTurnAdditionalTimesPower`:

```java
            AbstractMonster m = (AbstractMonster)action.target;

            for (int i = 0; i < amount; ++i) {
                CardMetaUtils.playCardAdditionalTime(card, m);
            }
```